### DISCUSSION TITLE
Icon request: fa-activitypub

### DISCUSSION THREAD

**Discussion by Aonrud** [FortAwesome/Font-Awesome#21183] [Icon Requests]
### Use case

It would be useful to have the generic ActivityPub logo, for example for linking to social media accounts on websites when the account isn't hosted on Mastodon.

_Note: Issue #16252 requests an icon for the Fediverse. As the Fediverse has no 'official' logo, and can include other protocols, I don't think this duplicates that request_ 

### Example image

This is the ActivityPub logo, as used on <https://activitypub.rocks/>, or see also [the Wikipedia article](https://en.wikipedia.org/wiki/ActivityPub).

Image from Wikimedia:
![ActivityPub logo](https://upload.wikimedia.org/wikipedia/commons/f/f2/ActivityPub-logo-symbol.svg)

### Icon request checklist

- [X] The title starts with "Icon request: " and is followed by the requested icon name
- [X] This icon does not represent a brand
- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The request is for a concrete object, or I've included an example image
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have [understood how requests work](https://fontawesome.com/community/leaderboard/new#faqs)

**Comment by Neustradamus** [comment]
At all, there is Fediverse too:
- https://github.com/FortAwesome/Font-Awesome/issues/16252

Please vote too.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
